docs/features/tenancy.md

```diff
@@ -1,8 +1,9 @@
-* IMPORTANT: This is an experimental feature supported only on standalone ESX as of release 0.12. We are seeking feedback/requirements to support mult-tenancy across vSphere Cluster/Datacenter. You can drop us a note at containers@vmware.com.
+---
```

Comment:
It seems that you delete the paragraph that "multi-tenancy is a experimental feature".